frontend/src/tests/components/eMIB/Inbox.test.js

```diff
@@ -1,10 +1,9 @@
 import React from "react";
 import { mount } from "enzyme";
-import Inbox from "../../../components/eMIB/Inbox";
-import Email from "../../../components/eMIB/Email";
-import { getInboxContent, getInboxLength } from "../../../components/eMIB/Emib";
+import { UnconnectedInbox as Inbox } from "../../../components/eMIB/Inbox";
```

Comment:
ok, THAT is cool. The unconnected export so it can be imported with a generic set of emails.